vite.config.js

```diff
@@ -19,13 +19,9 @@ function LocaleHotReload () {
 
 /** @type {import('vite').UserConfig} */
 const config = {
-    plugins: [sveltekit(), LocaleHotReload()],
+    // Completely disabled locale hot reload to fix infinite HMR refresh loops
```

Comment:
This is key local development functionality; it looks like you were running into a defect in it? If so, please open an issue to specify the defect, so we can resolve and restore this local development functionality.